services/tables/src/main/java/com/linkedin/openhouse/tables/api/handler/TablesApiHandler.java

```diff
@@ -61,14 +61,17 @@ ApiResponse<GetTableResponseBody> updateTable(
       String tableCreatorUpdator);
 
   /**
-   * Function to Delete a Table Resource identified by tableId in a given databaseId
+   * Function to Delete a Table Resource identified by tableId in a given databaseId with option to
```

Comment:
can the first iteration of this code be without changing the client side code? and instead having a whitelist of schema loaded in runtime into server that will be used in dropTable method?